tests/test_bgp_af.rb

```diff
@@ -314,65 +371,59 @@ def test_additional_paths_selection
     #
     # Set and verify
     #
+    if platform == :ios_xr
+      config_ios_xr_dependencies(asn)
+      config('route-policy drop_all', '#drop_all', 'end-policy')
```

Comment:
Why not move this into config_ios_xr_dependencies(asn)?

You can look at bgp neighbor af for similar:
  def setup_ios_xr
...
 # Global requirements
    # route policies
    str1 = 'route-map-in-name'
    str2 = 'route-map-out-name'
    config("route-policy #{str1}", 'end-policy')
    config("route-policy #{str2}", 'end-policy')
    config("route-policy #{str1.reverse}", 'end-policy')
    config("route-policy #{str2.reverse}", 'end-policy')
    config('route-policy foo_bar', 'end-policy')
    config('route-policy baz_inga', 'end-policy')